.github/fabricbot.json

```diff
@@ -2864,6 +2864,138 @@
     }
   },
   {
+    "taskSource": "fabricbot-config",
```

Comment:
The diff isn't particularly illuminating, what updates are being made to pod automation in this case?